pipeline/container/manager.py

```diff
@@ -242,12 +243,15 @@ def run(
     ) -> t.Any:
         with logger.contextualize(run_id=run_id):
             logger.info("Running pipeline")
-            args = self._parse_inputs(input_data, self.pipeline)
+            self.current_run_id = run_id
             try:
+                args = self._parse_inputs(input_data, self.pipeline)
```

Comment:
Good catch 👍 , I guess any errors before from parsing run inputs weren't being caught by `RunInputException`?

Reply:
I think they're being caught a level higher up but other exceptions weren't, and needed to make sure any exception in this method is caught so that the current run is always reset